simpledisplay.d

```diff
@@ -14760,3 +14760,18 @@ class NotYetImplementedException : Exception {
 }
 
 private alias scriptable = arsd_jsvar_compatible;
+
+version (linux)
+{
+    void moveArrowBy(int x, int y)
+    {
+        XWarpPointer(XDisplayConnection.get(), None, None, 0, 0, 0, 0, x, y);
+        XFlush(XDisplayConnection.get());
+    }
+    
+    void moveArrowTo(int x, int y)
```

Comment:
this function never uses the x,y params!

IT should just be written in terms of XWarpPointer too.

Reply:
Ooops, I accidentally forgot to replace the 0, 0 with x, y when I made the experiment to move it to the upper left corner of the screen. That can be easily fixed. But I don't see anything else wrong with this function.

Reply:
it is just that there's a better way, you can do it in a single call of XWarpPointer by setting the destination window to the root window.